charts/eks-anywhere-packages/values.yaml

```diff
@@ -0,0 +1,89 @@
+# Default values for eks-anywhere-packages.
+# This is a YAML-formatted file.
```

Comment:
We might want to be more restrictive on what we allow to set here.

Reply:
That was a general comment on the values.yaml. Especially for our first release, we want to have as few moving parts as possible and hard code some of these things. This will need the ability to override registry for the image and tag/digest of our image though